src/main/kotlin/no/nav/familie/ef/søknad/api/dto/søknadsdialog/Medlemskap.kt

```diff
@@ -3,10 +3,12 @@ package no.nav.familie.ef.søknad.api.dto.søknadsdialog
 data class Medlemskap(
     val perioderBoddIUtlandet: List<PerioderBoddIUtlandet>? = listOf(),
     val søkerBosattINorgeSisteTreÅr: BooleanFelt,
+    val oppholdsland: TekstFelt?,
```

Comment:
I tilfelle gammel frontend
```suggestion
    val oppholdsland: TekstFelt? = null,
```